VikingQuiz.Api/Utilities/UserExtensions.cs

```diff
@@ -1,16 +1,16 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
 using System.Security.Claims;
-using System.Threading.Tasks;
+using System.Security.Cryptography;
+using System.Text;
 
 namespace VikingQuiz.Api.Utilities
 {
     public static class UserExtensions
     {
-        public static int GetUserId(this ClaimsPrincipal user)
+        public static int GetUserId(this IEnumerable<Claim> user)
         {
-            var strId = user.Claims.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier).Value;
+            var strId = user.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier).Value;
```

Comment:
Try to use first letter of the entity name in lambda expressions (i.e. u instead of x)